float64 X_max
float64 X_min
float64 theta_torso
---
float64 X_target

